openedx/features/calendar/models.py

```diff
@@ -0,0 +1,20 @@
+"""
+Models for Calendar
+"""
+
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from opaque_keys.edx.django.models import CourseKeyField
+
+
+class UserCalendar(models.Model):
+    """
+    Model to track if a user has the calendar integration enabled for a specific Course
+
+    .. no_pii:
+    """
+    user = models.ForeignKey(User, db_index=True, on_delete=models.CASCADE)
+    course_key = CourseKeyField(max_length=255, db_index=True)
+    enabled = models.BooleanField(default=False)
```

Comment:
Maybe worth adding a utility method like `is_enabled_for_course(user, course)` that does a `get` with a default value of False if it doesn't exist in the model?